boilerplate/app/models/helpers/useStores.ts

```diff
@@ -57,11 +56,13 @@ export const useInitialRootStore = (callback: () => void | Promise<void>) => {
     let _unsubscribe
     ;(async () => {
       // set up the RootStore (returns the state restored from AsyncStorage)
-      const { restoredState, unsubscribe } = await setupRootStore(rootStore)
+      const { unsubscribe } = await setupRootStore(rootStore)
```

Comment:
Is restoredState no longer needed?

Reply:
Yeah, it was being used in the setupStore function to log the initial state to Reactotron: https://github.com/infinitered/ignite/pull/2465/files#diff-bcccc60bab0fb409a76223075a440e6b5f004da8e68734b92a8e4dbb0161bd0aL69

We could add this back, but I felt it was nice to reduce the noise a bit